keywords/SyncGateway.py

```diff
@@ -220,13 +220,22 @@ def start_sync_gateways(self, cluster_config, url=None, config=None):
             "server_scheme": self.server_scheme,
             "autoimport": "",
             "xattrs": "",
+            "no_conflicts": "",
+            "revs_limit": "",
             "couchbase_server_primary_node": couchbase_server_primary_node
         }
 
         if is_xattrs_enabled(cluster_config):
             playbook_vars["autoimport"] = '"import_docs": "continuous",'
             playbook_vars["xattrs"] = '"enable_shared_bucket_access": true,'
 
+        if no_conflicts_enabled(cluster_config):
+            playbook_vars["no_conflicts"] = '"unsupported": { "allow_conflicts": false },'
+        try:
+            revs_limit = get_revs_limit(cluster_config)
+            playbook_vars["revs_limit"] = '"revs_limit": {},'.format(revs_limit)
+        except KeyError as ex:
+            log_info("Keyerror in getting revs_limit{}".format(ex.message))
```

Comment:
Here , i should not raise exception here. If revs_limit is not existed in cluster_config, it should add revs_limit in config, To avoid exception of keyerror, I need to catch here and let go beyond and should not kill the test